### PR TITLE
Correct outdated links and info in argocd docs.

### PR DESCRIPTION
**Pull request by HumairAK**
Follow up on : https://github.com/operate-first/support/pull/432

### CODE REVIEW

File: docs/argocd-gitops/onboarding_to_argocd.md [hunk 1]
```diff
@@ -1,33 +1,52 @@
...
 
-Each team is given an [ArgoCD Project][2] that has an allow-list of clusters and namespaces to which they can deploy. They also have an allow-list of resources they can deploy within those namespaces. These restrictions exist to prevent teams from being able to use ArgoCD to deploy cluster scoped resources, or resources onto other team's namespaces. You can browse the projects [here][3] to get an idea of the current set of permissions. Note that for most projects, majority of these permissions are inherited from the [global project][4].
+The ArgoCD instance is deployed on [MOC Infra][1] cluster.
+
+Each team is given an [ArgoCD Project][2] that has an allow-list of clusters and namespaces to which they can deploy.
+They also have an allow-list of resources they can deploy within those namespaces. These restrictions exist to prevent
+teams from being able to use ArgoCD to deploy cluster scoped resources, or resources onto other team's namespaces. You
+can browse the projects [here][3] to get an idea of the current set of permissions. Note that for most projects,
+majority of these permissions are inherited from the [global project][4].
```
Comment (Gregory-Pereira) [inline docs/argocd-gitops/onboarding_to_argocd.md:11]: Should we add something about the process one would go through if they wanted to adjust these permissions, such as if they were deploying an application that required them to enable a new type of CRD in the global project?

**Review by Gregory-Pereira — APPROVED** [review]
/Igtm all good 👍

File: docs/argocd-gitops/onboarding_to_argocd.md (hunk 1 above)
Comment (HumairAK) [reply docs/argocd-gitops/onboarding_to_argocd.md:11]: Like this? https://github.com/operate-first/apps/blob/master/docs/argocd-gitops/add_permissions_to_project.md

Comment (Gregory-Pereira) [reply docs/argocd-gitops/onboarding_to_argocd.md:11]: Yeah, good to know its already addressed.

File: docs/argocd-gitops/onboarding_to_argocd.md [hunk 2]
```diff
@@ -42,6 +61,8 @@ spec:
   destinations:
     - namespace: '<team-namespace-prefix>-*'
       server: '<cluster-name>'
+    - namespace: '<namespace-name>' # if there is no generic team name prefix, then specify namespaces individually
+      server: '<cluster-name>'
```
Comment (famartinrh) [inline docs/argocd-gitops/onboarding_to_argocd.md:65]: should be `name` , or the description for `<cluster-name>` should change

Comment (HumairAK) [reply docs/argocd-gitops/onboarding_to_argocd.md]: good catch, updated!

**Comment by HumairAK** [comment]
/hold 

for: https://github.com/operate-first/apps/pull/1366

**Comment by HumairAK** [comment]
Let's merge it actually, will rebase the other pr

**Pull request merged**